File: app/src/main/java/com/tvm/doctorcube/settings/FragmentPrivacy.java
package com.tvm.doctorcube.settings;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;
import androidx.appcompat.widget.Toolbar;
import androidx.fragment.app.Fragment;
import androidx.navigation.NavController;
import androidx.navigation.fragment.NavHostFragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import com.tvm.doctorcube.R;

public class FragmentPrivacy extends Fragment {

    private NavController navController;
    private Toolbar toolbar;

    public FragmentPrivacy() {
        // Required empty public constructor
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        // Inflate the layout for this fragment
        View view = inflater.inflate(R.layout.fragment_privacy, container, false);

        // Initialize NavController
        navController = NavHostFragment.findNavController(this);

        // Hide MainActivity Toolbar

        // Initialize Toolbar

        return view;
    }



    @Override
    public void onDestroyView() {
        super.onDestroyView();
        // Show MainActivity Toolbar when leaving this Fragment
    }

    @Override
    public void onDetach() {
        super.onDetach();
        if (navController != null) {
            navigateToHome();
        }
    }

    private void navigateToHome() {

    }
}

File: app/src/main/java/com/tvm/doctorcube/settings/FragmentPrivacy.kt
package com.tvm.doctorcube.settings

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import androidx.navigation.NavController
import androidx.navigation.fragment.NavHostFragment
import com.tvm.doctorcube.databinding.FragmentPrivacyBinding

class FragmentPrivacy : Fragment() {
    private var navController: NavController? = null
    private lateinit var binding: FragmentPrivacyBinding

    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {

        binding= FragmentPrivacyBinding.inflate(inflater,container,false)
        // Inflate the layout for this fragment
        val view = binding.root

        // Initialize NavController
        navController = NavHostFragment.findNavController(this)


        // Initialize Toolbar
        return view
    }


    override fun onDetach() {
        super.onDetach()
        if (navController != null) {
            navigateToHome()
        }
    }

    private fun navigateToHome() {
    }
}